_pages/ato/types.md

```diff
@@ -3,17 +3,11 @@ title: Types of ATO
 navtitle: Types
 ---
 
-Choosing an authorization is an important decision. Work with your Infrastructure Lead and Product Owner to determine which authorization you will employ, and when. The Infrastructure Lead makes all final decisions and answers in writing (email, Slack, etc) any questions the team may have on how to meet the below objectives. Many situations may have no "right answer". It is up to the Infrastructure Lead to make a risk-based decision on what to do.
+In most cases, the types of ATO that will be pursued for GSA systems are the *GSA Lightweight ATO (LATO)*. The GSA LATO process is described in a guide on [Insite](https://insite.gsa.gov/portal/content/627230) (search for "Lightweight Security Authorization Process" on that page). Systems that are under development can be operated without an ATO provided they fulfill the requirements for [pre-assessment for internal government use](#conditions-for-pre-assessment).
 
-In most cases, the two types of ATO that will be pursued for GSA systems are the *GSA Lightweight ATO (LATO)* and the *FedRAMP Agency ATO*. Systems that are under development can be operated without an ATO provided they fulfil the requirements for [pre-assessment for internal government use](#conditions-for-pre-assessment).
+The GSA LATO is designed for Low and Moderate impact [level](../levels/) systems built using agile methods that run on top of cloud infrastructure which has already received an ATO (such as AWS, Azure, and [cloud.gov](https://cloud.gov)). It's "lightweight" because only 24 NIST [controls](../controls/) must be implemented in order to grant this type of ATO. The GSA LATO is a good option for systems 18F intends to own and operate.
```

Comment:
> The GSA LATO is a good option for systems 18F intends to own and operate.

I would delete this sentence...I don't think this has been a deciding factor about whether to do a LATO or not.